netket/vqs/mc/mc_state/state.py

```diff
@@ -549,6 +550,50 @@ def log_value(self, σ: jnp.ndarray) -> jnp.ndarray:
         """
         return jit_evaluate(self._apply_fun, self.variables, σ)
 
+    def local_estimators(self, op: AbstractOperator, *, chunk_size: int = None):
+        r"""
+        Compute the local estimators for the operator :code:`op` (also known as local energies
+        when :code:`op` is the Hamiltonian) at the current configuration samples :code:`self.samples`.
+
+        .. math::
+
+            O_\mathrm{loc}(s) = \frac{\langle s | \mathtt{op} | \psi \rangle}{\langle s | \psi \rangle}
+
+        .. warning::
+
+            The samples differ between MPI processes, so returned the local estimators will
+            also take different values on each process. To compute sample averages and similar
+            quantities, you will need to perform explicit operations over all MPI ranks.
+            (Use functions like :code:`self.expect` to get process-independent quantities without
+            manual reductions.)
+
+        Args:
+            op: The operator.
+            chunk_size: Suggested maximum size of the chunks used in forward and backward evaluations
+                of the model. (Default: :code:`self.chunk_size`)
+        """
+        s, (sp, mels) = get_local_kernel_arguments(self, op)
```

Comment:
It's not part of the `get_local_kernel_arguments` contract that `extra_args` are this tuple
```suggestion
        s, extra_args = get_local_kernel_arguments(self, op)
```